ajax/libs/openseadragon/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "openseadragon",
+  "version": "2.2.1",
+  "filename": "openseadragon.min.js",
+  "description": "Provides a smooth, zoomable user interface for HTML/Javascript.",
+  "keywords": [
+    "image",
+    "zoom",
+    "pan",
+    "openseadragon",
+    "seadragon",
+    "deepzoom",
+    "dzi",
+    "iiif",
+    "osm",
+    "tms"
+  ],
+  "homepage": "https://openseadragon.github.io/",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/openseadragon/openseadragon.git"
+  },
+  "author": {
```

Comment:
It is added by myself. I traced the commit contributor, and all members of openseadragon contribute to this project.
https://github.com/openseadragon/openseadragon/graphs/contributors